cupy/manipulation/add_remove.py

```diff
@@ -12,8 +13,62 @@
 
 # TODO(okuta): Implement resize
 
+_first_nonzero_krnl = core.ReductionKernel(
+    'raw S data, T indices, T len',
```

Comment:
`indices` is not needed anymore. `len` is always of int64 type.
```suggestion
    'raw S data, int64 len',
```